packages/README.md

```diff
@@ -256,3 +256,108 @@ Ensure that the `build-types` directory will be included in the published packag
 
 [lerna]: https://lerna.js.org/
 [npm]: https://www.npmjs.com/
+
+## Side effects
```

Comment:
Should it start with something more general like:
`## Optimizing for bundlers`?

Reply:
Done!